packages/angular_devkit/build_angular/src/builders/browser/tests/options/named-chunks_spec.ts

```diff
@@ -61,6 +61,7 @@ describeBuilder(buildWebpackBrowser, BROWSER_BUILDER_INFO, (harness) => {
       const { result } = await harness.executeOnce();
 
       expect(result?.success).toBe(true);
+      debugger;
```

Comment:
```suggestion

```

Reply:
Haha, yeah, wasn't added by me, but I will fix it!